ledgerwallet/transport/__init__.py

```diff
@@ -9,3 +10,17 @@ def enumerate_devices():
     for cls in DEVICE_CLASSES:
         devices.extend(cls.enumerate_devices())
     return devices
+
+
+class open_device:
+    """Open a device in a context manager."""
+
+    def __init__(self, device: Device):
+        self.device = device
+
+    def __enter__(self):
+        self.device.open()
+        return self.device
+
+    def __exit__(self, *exc_details):
+        self.device.close()
```

Comment:
I find it cleaner to use the context manager decorator from [contextlib](https://docs.python.org/3/library/contextlib.html):

```python
@contextmanager
def open_device(dev: Device):
    try:
        dev.open()
        yield dev
    finally:
        dev.close()
```

What do you think?

Reply:
I agree, just updated the PR